lib/internal/process/next_tick.js

```diff
@@ -88,20 +109,58 @@ function setupNextTick() {
     }
   }
 
+  // TODO(trevnorris): Using std::stack is much slower here than was the
+  // Float64Array stack used in a previous implementation. Problem is the
+  // Float64Array stack was a bit brittle. Investigate how to harden that
+  // implementation and possibly reintroduce it.
```

Comment:
It's referring to `Environment::AsyncHooks::ids_stack_`. My old implementation had an overly complex implementation to track the stack of execution contexts. It was faster, but also brittle and difficult to understand. Moved to using `std::stack` to simplify things, but it's slower. The `TODO` is a reminder to investigate going back to a similar implementation as the previous, but not so badly written.